backend/app/tests/utils/conversation.py

```
@@ -0,0 +1,15 @@
+import secrets
+import string
+from sqlmodel import Session, select
```

Ruff (0.12.2)

`sqlmodel.Session` imported but unused

Remove unused import

(F401)